backend/molgenis-emx2-rdf/src/main/java/org/molgenis/emx2/rdf/RDFService.java

```diff
@@ -529,12 +531,6 @@ private void dataRowToRdf(
     }
     builder.add(subject, IRI_DATASET_PREDICATE, tableIRI);
     builder.add(subject, RDFS.LABEL, Values.literal(getLabelForRow(row, table.getMetadata())));
-    // via rowId might be subclass
-    if (rowId != null) {
-      // because row IRI point to root tables we need to find actual subclass table to ensure we
-      // get all columns
-      table = getSubclassTableForRowBasedOnMgTableclass(table, row);
-    }
```

Comment:
Should not be needed due to:
https://github.com/molgenis/molgenis-emx2/blob/cb19bed6e94be5e64bff152c19cf3205095d9475/backend/molgenis-emx2-rdf/src/main/java/org/molgenis/emx2/rdf/RDFService.java#L213-L236
(this is done before any rowId filter is done on a per-table level)

See https://github.com/molgenis/molgenis-emx2/pull/4831#issue-2916677556 for existing test that should validate the behaviour.